src/config/dynamoInit.js

```diff
@@ -0,0 +1,12 @@
+import AWS from 'aws-sdk';
+
+export const initialDynamo = () => {
+    const { AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, PORT } = process.env;
+    if (!AWS_ACCESS_KEY_ID || !AWS_SECRET_ACCESS_KEY || !PORT) {
+        throw new Error('AWS_ACCESS_KEY, SECRET_KEY or PORT is not set');
+    }
+    AWS.config.update({
```

Comment:
this has nothing to do with dynamo DB, this is just configuring AWS to with the key and secret. The function name should probably change to something else.
also why is this function returning `PORT`?